packages/material-ui/src/CssBaseline/CssBaseline.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { exactProp } from '@material-ui/utils';
-import withStyles from '../styles/withStyles';
+import useThemeProps from '../styles/useThemeProps';
+import Global from '../Global';
```

Comment:
What are your thoughts on naming these components `GlobalStyle` instead of `Global`?

Reply:
`GlobalStyles` is better for a neme. I choose `Global' because that is the name emotion uses, but in their context it makes sense, in our I think `GlobalStyles` is better. Will rename it in all three packages for consistency.

Reply:
Done.